tests/frontend/test_kibana_app/cypress/integration/features/management/settings/sample-data.feature

```diff
@@ -0,0 +1,31 @@
+Feature: add sample data to modules
+
+  As a kibana user
+  i want to add sample data to modules
+  in order to populate graphics
+
+  Scenario: Add sample security information data
+    Given The kibana admin user is logged in using odfe authentication
+    When The user navigates to Sample data settings
+    And The user adds sample security information data
+    Then The confirmation toast is displayed
+
+  Scenario: Add sample auditing and policy monitoring data
+    Given The kibana admin user is logged in using odfe authentication
+    When The user navigates to Sample data settings
+    And The user adds sample auditing and policy monitoring data
+    Then The confirmation toast is displayed
+
+  Scenario: Add sample threat detection and response data
+    Given The kibana admin user is logged in using odfe authentication
+    When The user navigates to Sample data settings
+    And The user adds sample threat detection and response data
+    Then The confirmation toast is displayed
+
+    Scenario: Delete all sample data
+      Given The kibana admin user is logged in using odfe authentication
+      When The user navigates to Sample data settings
+      And The user removes sample security information data
+      And The user removes sample auditing and policy monitoring data
+      And The user removes sample threat detection and response data
+      Then The confirmation toast is displayed
```

Comment:
I think is better with a table to check for each success message
```cucumber
When The user navigates to Sample
And The user removes this sample data
| Security information |
| Auditing and policy monitoring | 
| Threat detection and response | 
Then The confirmation toast is displayed


```

Reply:
great Idea!